From 84320668989f5c8988e92c0d98df5e482720e11d Mon Sep 17 00:00:00 2001
From: "Joshua M. Clulow" <jmc@joyent.com>
Date: Thu, 15 Sep 2016 08:00:04 +0000
Subject: [PATCH] HEAD-2321 iPXE should wait longer for DHCP responses Reviewed
 by: Cody Mello <melloc@joyent.com> Reviewed by: Robert Mustacchi
 <rm@joyent.com> Approved by: Robert Mustacchi <rm@joyent.com>

---
 tftpboot/undionly.kpxe | Bin 79065 -> 72697 bytes
 1 file changed, 0 insertions(+), 0 deletions(-)

diff --git a/tftpboot/undionly.kpxe b/tftpboot/undionly.kpxe
index f5309ba..2fd8133 100644
Binary files a/tftpboot/undionly.kpxe and b/tftpboot/undionly.kpxe differ
-- 
2.21.0

